docs/20_tutorial.md

```diff
@@ -2,3 +2,63 @@
 
 Tutorial
 ========
+
+***should the tutorial deal with the SOS establishing the risk
+limit, or any of their pre-audit responsibilities like the selection
+of which races to audit?
+
+Brief introduction to RLAs in non-technical, human friendly language.
+At the county level, something along the lines of "This tool is going
+to help you cross reference votes recorded on paper ballots with how 
+the computers read those ballots, to determine whether the computers
+did their job accurately…” Try to put their minds at ease about the
+process. :)
+
+***should the tutorial cover the mechanics of storing ballots and the
+other physical aspects of the RLA process which are not ‘in’ the RLA
+tool/system?
+
+Basic overview: Screenshots of the dashboard/tool, ordered in the sequence
+they will be encountered, with brief, clear instructions describing what
+is expected of the user(s) at each step, starting with the login screen
+and proceeding through the completion of the audit, or determination that
+a full hand count is required. First SOS, then County, then Public.
+
+Detailed instructions with associated screenshot about the process of
+Audit Board members signing in.
+
+Detailed descripton of the SOS dashboard with associated screenshot with
+descriptions of all fields and/or information/updates.
+
+Detailed description with associated screenshot of the County dashboard
+with instructions about the process of uploading ballot manifests and
+CVRs, and associated hash functions.
+
+***should the tutorial cover the preparation/creation of these artifacts
+they’ll be uploading?
+
+Description of the process of generating the random seed, perhaps with a
+photograph of what the process will look like with folks looking on.
```

Comment:
In particular, clarity that the manifests and hashes of the CVRs must be published before the rolling of the dice.
I hope that the tool takes care of that via not just making them available via the public interface, but also obtaining a trustworthy timestamp of all or each of those artifacts verifiable by people after-the-fact. E.g. via Guardtime timestamps.